release_bot/releasebot.py

```diff
@@ -81,9 +81,24 @@ def find_open_release_issues(self):
             else:
                 for edge in reversed(edges):
                     cursor = edge['cursor']
-                    match = re.match(r'(.+) release', edge['node']['title'].lower())
+                    last_version = Version(self.github.latest_release())
+                    version = ''
+                    match = False
+                    re_match = re.match(r'(.+) release', edge['node']['title'].lower())
+                    if re_match:
+                        match = True
+                        version = re_match[1].strip()
+                    elif edge['node']['title'].lower().strip() == "new major release":
+                        match = True
+                        version = str(last_version.next_major())
+                    elif edge['node']['title'].lower().strip() == "new minor release":
+                        match = True
+                        version = str(last_version.next_minor())
+                    elif edge['node']['title'].lower().strip() == "new patch release":
```

Comment:
Can we store `edge['node']['title'].lower().strip()` into a variable so we don't repeat it several times?